packages/loot-core/src/types/server-handlers.d.ts

```diff
@@ -275,9 +273,9 @@ export interface ServerHandlers {
 
   'get-did-bootstrap': () => Promise<boolean>;
 
-  'subscribe-needs-bootstrap': (
-    args: { url } = {},
-  ) => Promise<
+  'subscribe-needs-bootstrap': (args: {
+    url;
+  }) => Promise<
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I wonder if there is some TS/eslint config we could enable to catch these.. 🤔